test-suite/squidconf/bad-acl-dstdomain-dupe.conf.instructions

```diff
@@ -1,25 +1,31 @@
 expect-messages <<END
 WARNING: Ignoring example.com because it is already covered by .example.com
-    advice: Remove value example.com from the ACL named test11
+    advice: Remove value example.com from the ACL
+    acl name: test11
 
-WARNING: Ignoring earlier example.com because it is covered by .example.com
-    advice: Remove value example.com from the ACL named test12
+WARNING: Ignoring earlier example.org because it is covered by .example.org
+    advice: Remove value example.org from the ACL
+    acl name: test12
 
 WARNING: Ignoring a.example.com because it is already covered by .example.com
-    advice: Remove value a.example.com from the ACL named test21
+    advice: Remove value a.example.com from the ACL
+    acl name: test21
 
 WARNING: Ignoring earlier b.example.com because it is covered by .example.com
-    advice: Remove value b.example.com from the ACL named test22
+    advice: Remove value b.example.com from the ACL
+    acl name: test22
 
 WARNING: Ignoring .example.com because it is already covered by .example.com
-    advice: Remove value .example.com from the ACL named test31
-
+    advice: Remove value .example.com from the ACL
 WARNING: Ignoring c.example.com because it is already covered by .example.com
-    advice: Remove value c.example.com from the ACL named test31
+    advice: Remove value c.example.com from the ACL
+    acl name: test31
```

Comment:
We managed to keep ACL names in all these checking patterns except when the ACL name is _repeated_ (because there are multiple warnings for the same ACL X, each containing the expected "acl name: X" line). In those cases, test-squid-conf.sh lacks code to check that there are _two_ "acl name: X" lines. There are some [hairy shell tricks](https://unix.stackexchange.com/questions/335276/grep-v-how-to-exclude-only-the-first-or-last-n-lines-that-match) that would allow us to implement those checks, but I do not think we should complicate test-squid-conf.sh so much. This problem is best solved after/if we migrate test-squid-conf.sh to a better scripting language.